tests/issue-53.cx

```
@@ -2,4 +2,5 @@ package main
 
 func main()() {
 	var i i32 = 4 + * f32.i32(5.0)
```

What does this mean?

---

Hahah, I also had the same thought. I guess @asahi3g was just testing that using multiple operands that way should throw an error (which now makes me think, we have to test for other combinations!). But! I then thought that that was `*f32.i32(5.0)`, so it checks to see if it can de dereferenced.